<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.554">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>R/Medicine 2024</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">R/Medicine 2024</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Register.html"> 
<span class="menu-text">Register</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Abstracts.html"> 
<span class="menu-text">Abstracts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Program.html"> 
<span class="menu-text">Program</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./workshops.html" aria-current="page"> 
<span class="menu-text">Workshops</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Attend.html"> 
<span class="menu-text">Code of Conduct</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Competition.html"> 
<span class="menu-text">Competition</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Contact.html"> 
<span class="menu-text">Contact us</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Events.html"> 
<span class="menu-text">Past Events</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./committees.html"> 
<span class="menu-text">Committees</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#workshops" id="toc-workshops" class="nav-link active" data-scroll-target="#workshops">Workshops</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">




<section id="workshops" class="level2">
<h2 class="anchored" data-anchor-id="workshops">Workshops</h2>
<div class="columns">
<div class="column" style="width:45%;">
<section id="lucy-dagostino-mcgowan" class="level3">
<h3 class="anchored" data-anchor-id="lucy-dagostino-mcgowan">Lucy D’Agostino McGowan</h3>
<p><img src="images/Lucy_D_M.jpg" class="img-fluid"></p>
</section>
<section id="biography" class="level3">
<h3 class="anchored" data-anchor-id="biography">Biography</h3>
<p><a href="https://www.lucymcgowan.com/about/">Lucy D’Agostino McGowan</a> is an assistant professor in the Department of Statistical Sciences at Wake Forest University. She received her PhD in Biostatistics from Vanderbilt University and completed her postdoctoral training at Johns Hopkins University Bloomberg School of Public Health. Her research focuses on analytic design theory, statistical communication, causal inference, and data science pedagogy.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="causal-inference-in-r" class="level2">
<h2 class="anchored" data-anchor-id="causal-inference-in-r">Causal Inference in R</h2>
<section id="abstract" class="level3">
<h3 class="anchored" data-anchor-id="abstract">Abstract</h3>
<p>This workshop will use the NHANES Epidemiologic Follow-up Study (NHEFS) data. In this workshop, we’ll teach the essential elements of answering causal questions in R through causal diagrams, and causal modeling techniques such as propensity scores and inverse probability weighting.</p>
</section>
<section id="dr.-mcgowan-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="dr.-mcgowan-on-the-web">Dr.&nbsp;McGowan on the Web</h3>
<p>Dr.&nbsp;McGowan can be found blogging at <a href="https://livefreeordichotomize.com/">livefreeordichotomize.com</a>, on Twitter <a href="https://twitter.com/LucyStats">@LucyStats</a>, and podcasting on the American Journal of Epidemiology partner podcast, <a href="https://casualinfer.libsyn.com/">Casual Inference</a>.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="joy-payton" class="level3">
<h3 class="anchored" data-anchor-id="joy-payton">Joy Payton</h3>
<p><img src="images/Payton.jpeg" class="img-fluid"></p>
</section>
<section id="biography-1" class="level3">
<h3 class="anchored" data-anchor-id="biography-1">Biography</h3>
<p><a href="https://www.linkedin.com/in/joypayton/">Joy Payton</a> is the Supervisor of Data Education at the Children’s Hospital of Philadelphia, where her goal is to make every scientist a data scientist. She has earned a Masters degree in Data Science from the City University of New York (CUNY), an Information Systems Security certificate from Penn State University, and her undergraduate degree at Agnes Scott College (Go Scotties!). She is a Google Cloud Certified Associate Cloud Engineer and an AWS Certified Cloud Practitioner.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="google-bigquery-first-steps-in-r" class="level2">
<h2 class="anchored" data-anchor-id="google-bigquery-first-steps-in-r">Google BigQuery: First Steps in R</h2>
<section id="abstract-1" class="level3">
<h3 class="anchored" data-anchor-id="abstract-1">Abstract</h3>
<p>In this hands-on workshop, you’ll use the free tier of Google Cloud Platform to work with large public datasets in Google’s cloud data warehousing solution, BigQuery. We’ll cover topics including:</p>
<ul>
<li><p>Brief overview of cloud solutions and why they matter in medicine</p></li>
<li><p>How to sign up for services for free for your own learning</p></li>
<li><p>Practicing SQL queries in BigQuery</p></li>
<li><p>Using R and BigQuery together</p></li>
</ul>
<p>You’ll leave this workshop with the beginnings of a data analysis of a large public dataset, a Google Cloud account that will allow you to continue learning on your own, and a newfound confidence in SQL, cloud computing, and system interoperability.</p>
</section>
<section id="joy-payton-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="joy-payton-on-the-web">Joy Payton on the Web</h3>
<p>Joy Payton can be found on Twitter <a href="https://twitter.com/kjoypayton">@KJoyPayton</a>, and sharing educational resources on the <a href="https://education.arcus.chop.edu/team/">Arcus Website</a>.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="garrick-aden-buie" class="level3">
<h3 class="anchored" data-anchor-id="garrick-aden-buie">Garrick Aden-Buie</h3>
<p><img src="images/garrick-profile.jpg" class="img-fluid"></p>
</section>
<section id="biography-2" class="level3">
<h3 class="anchored" data-anchor-id="biography-2">Biography</h3>
<p><a href="https://www.garrickadenbuie.com/">Garrick Aden-Buie</a> is an software engineer for Shiny at Posit. He received a BS from Lehigh University in Applied Mathematics, and developed smart home sensors for healthy aging in place at the University of South Florida, before becoming a data scientist at Moffit Cancer Center. At Posit, Garrick builds tools that help everyone do data science in R with RMarkdown, Quarto, and Shiny.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="next-generation-shiny-apps-with-bslib" class="level2">
<h2 class="anchored" data-anchor-id="next-generation-shiny-apps-with-bslib">Next Generation Shiny Apps with {bslib}</h2>
<section id="abstract-2" class="level3">
<h3 class="anchored" data-anchor-id="abstract-2">Abstract</h3>
<p>Learn to build Shiny apps using modern user interfaces (UI) and layouts with bslib, the next generation of Shiny UI. We’ll cover stylish and convenient dashboard layouts and components as we showcase the ways in which bslib can replace shinydashboard. We’ll also learn about exciting new ways to deploy Shiny apps with shinylive as static sites that run entirely in the user’s browser and don’t require setting up or maintaining a Shiny server.</p>
</section>
<section id="garrick-aden-buie-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="garrick-aden-buie-on-the-web">Garrick Aden-Buie on the Web</h3>
<p>Garrick can be found blogging at <a href="https://www.garrickadenbuie.com/blog/">garrickadenbuie.com</a>, on Mastodon <a href="https://fosstodon.org/@grrrck">@grrrck@fosstodon.org</a>, and sticking things together with <a href="https://github.com/gadenbuie/epoxy">epoxy</a>.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="stephan-kadauke-and-will-beasley" class="level3">
<h3 class="anchored" data-anchor-id="stephan-kadauke-and-will-beasley">Stephan Kadauke and Will Beasley</h3>
<p><img src="images/kadauke_stephan.jpeg" class="img-fluid"></p>
</section>
<section id="biographies" class="level3">
<h3 class="anchored" data-anchor-id="biographies">Biographies</h3>
<p><a href="https://www.chop.edu/doctors/kadauke-stephan">Stephan Kadauke</a> is the Assistant Director of the Cell and Gene Therapy Laboratory at Children’s Hospital of Philadelphia. He received his MD degree and a PhD in Cell and Molecular Biology from the University of Pennsylvania, and did his residency in Clinical Pathology at Massachusetts General Hospital, before Fellowship in Transfusion Medicine at Harvard. At CHOP, he is responsible for the development and implementation of new technologies including CAR-T therapy in the Cell and Gene Therapy Laboratory.</p>
<p><img src="images/william_beasley.jpg" class="img-fluid"></p>
<p><a href="https://www.chop.edu/doctors/kadauke-stephan">Will Beasley</a> is a Professor of Pediatrics and leads the Clinical Research Data Warehouse at the Oklahoma University Health Sciences Center. He received his PhD in quantitative psychology from University of Oklahoma. In addition to his role in BBMC and the CRDW, he provides statistical analysis and software development to research studies at OUHSC. Dr.&nbsp;Beasley’s areas of expertise and interest include Bayesian and Frequentist statistics, bioinformatics, simulation methods, and exploratory and graphical analysis, and reproducible research. He also has a wealth of software programming experience including SQL Server, C#, and R.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="tidying-your-redcap-data-with-redcaptidier" class="level2">
<h2 class="anchored" data-anchor-id="tidying-your-redcap-data-with-redcaptidier">Tidying your REDCap Data with {REDCapTidieR}</h2>
<section id="abstract-3" class="level3">
<h3 class="anchored" data-anchor-id="abstract-3">Abstract</h3>
<p>This workshop will use the {REDCap Tidier} package to get clinical data out of REDCap and prepare it for analysis. REDCap is an electronic data capture software that is widely used in the medical research community. The REDCapR package streamlines calls to the REDCap API from an R environment. One of REDCapR’s main uses is to import records from a REDCap project. This works well for simple projects, however this workflow becomes fugly when complex databases that include longitudinal structure and/or repeating instruments are used.</p>
<p>The REDCapTidieR package aims to make the life of analysts who deal with complex REDCap databases easier. It builds upon REDCapR to make its output tidier. Instead of one large data frame that contains all the data from your project, you get to work with a set of tidy tibbles, one for each REDCap instrument.</p>
</section>
<section id="stephan-kadauke-and-will-beasley-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="stephan-kadauke-and-will-beasley-on-the-web">Stephan Kadauke and Will Beasley on the Web</h3>
<p>Stephan can be found automating everything at CHOP, on Twitter at <a href="https://twitter.com/StephanKadauke">@StephanKadauke</a>, and promoting reproducible medical research with R.</p>
<p>Will can be found working on improving <a href="https://ouhscbbmc.github.io/REDCapR/articles/workflow-read.html">REDCap Workflows</a>, and supporting a variety of research studies at OUHSC.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="catalina-canizares-escobar-and-francisco-cardozo" class="level3">
<h3 class="anchored" data-anchor-id="catalina-canizares-escobar-and-francisco-cardozo">Catalina Canizares-Escobar and Francisco Cardozo</h3>
<p><img src="images/catalina.jpeg" class="img-fluid"> <img src="images/francisco.jpeg" class="img-fluid"></p>
</section>
<section id="biography-3" class="level3">
<h3 class="anchored" data-anchor-id="biography-3">Biography</h3>
<p><a href="https://global.fiu.edu/world-centers/world-center-colombia/fulbrighters/catalina-canizares-escobar.html">Catalina Cañizares</a> is a passionate data scientist and a Ph.D.&nbsp;candidate in Social Welfare, dedicated to using data to gain insights into emotional disorders. She has been delving deep into data analysis, especially with R. Her focus? Making data understandable and useful. She specializes in cleaning and merging data, and loves exploring data with tools like tidyverse, table1, gtsummary, and skimr, among others. She is also interested in using Machine Learning models, with the tidymodels package, to better understand emotional disorders. Plus, She is all about keeping things clear and reproducible by using tools such as Quarto.</p>
<p><a href="https://focardozom.github.io/">Francisco Cardozo</a> is a PhD canidate in prevention science and community health, he specializes in applying quantitative techniques to evaluate the efficacy of prevention programs, focusing on understanding the dynamics of how and for whom these programs are most effective. He is dedicated to developing precise measurements and analyses that inform decisions about program operations. Francisco is passionate about translating resource science into practical, real-world applications.</p>
</section>
<section id="biografía" class="level3">
<h3 class="anchored" data-anchor-id="biografía">Biografía</h3>
<p><a href="https://global.fiu.edu/world-centers/world-center-colombia/fulbrighters/catalina-canizares-escobar.html">Catalina Cañizares</a> es una científica de datos apasionada y candidata Ph.D.&nbsp;en trabajo social. Ella se dedidca al uso de datos para obtener información sobre los trastornos emocionales. Ha estado profundizando en el análisis de datos, especialmente con R y su enfoque es hacer que los datos sean comprensibles y útiles. Se especializo en limpiar y fusionar datos, y le encanta explorar datos con herramientas como tidyverse, table1, gtsummary y skimr, entre otras. También se interesa utilizar modelos de Machine Learning, con el paquete tidymodels, para comprender mejor los trastornos emocionales.</p>
<p><a href="https://focardozom.github.io/">Francisco Cardozo</a> es un candidato a PhD en Ciencias de las Prevención y Salud Comunitaria, se especializa en aplicar técnicas cuantitativas para evaluar la eficacia de programas de prevención, centrándose en entender la dinámica de cómo y para quién estos programas son más efectivos. Está dedicado a desarrollar mediciones y análisis que informen decisiones sobre cómo operar los programas. Francisco siente pasión por traducir hallazgos científicos en aplicaciones prácticas.</p>
</section>
<section id="enhancing-scientific-equity-a-spanish-introduction-to-using-r-for-biostatistical-and-data-science-programming" class="level2">
<h2 class="anchored" data-anchor-id="enhancing-scientific-equity-a-spanish-introduction-to-using-r-for-biostatistical-and-data-science-programming">Enhancing Scientific Equity: A Spanish Introduction to Using R for Biostatistical and Data Science Programming</h2>
<section id="abstract-4" class="level3">
<h3 class="anchored" data-anchor-id="abstract-4">Abstract</h3>
<p>Despite the abundant resources available for learning R, most of these materials are primarily accessible to English speakers. This language barrier significantly restricts access for individuals who do not speak English proficiently. As a result, Spanish-speaking communities often face considerable challenges in accessing software training opportunities. This disparity leads to inequities in the distribution and utilization of scientific technologies, which is particularly concerning given the increasing importance of digital skills nowadays. To mitigate these challenges and promote inclusivity, we propose conducting a programming workshop in Spanish during the conference. This initiative aims to bridge the gap by providing Spanish-speaking participants with equal opportunities to engage with and benefit from technological advancements. By doing so, we not only enhance individual capabilities but also contribute to a more equitable distribution of educational resources in the scientific community. This workshop will equip attendees with basic skills in R. Our primary objective is to familiarize participants with RStudio and its key features for generating reproducible reports. We will guide attendees through the process of creating and managing projects in RStudio and introduce them to creating reproducible manuscripts using Quarto documents. The workshop will utilize a publicly available dataset from the CDC, which contains information on drug use and suicidal ideation among adolescents, as a practical example of using R for academic research in public health. We will explain how to use functions such as filter, mutate, summarize, and select from the tidyverse suite of packages. We will conclude by demonstrating how to use ggplot2 to create visualizations in R. By the end of the workshop, participants will have created a, reproducible document in HTML format, detailing the data cleaning steps and analysis of a significant, contemporary social issue. This presentation aims to close the gap in programming literacy among Spanish-speaking researchers and promote methods for reproducible scientific inquiry.</p>
</section>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="promover-la-equidad-científica-una-introducción-al-uso-de-r-para-la-programación-en-bioestadística-y-ciencia-de-datos-en-español." class="level2">
<h2 class="anchored" data-anchor-id="promover-la-equidad-científica-una-introducción-al-uso-de-r-para-la-programación-en-bioestadística-y-ciencia-de-datos-en-español.">Promover la Equidad Científica: Una Introducción al uso de R para la programación en Bioestadística y Ciencia de Datos, en Español.</h2>
<section id="resumen" class="level3">
<h3 class="anchored" data-anchor-id="resumen">Resumen</h3>
<p>A pesar de los abundantes recursos disponibles para aprender R, la mayoría de estos materiales son accesibles principalmente para angloparlantes. Esta barrera del idioma restringe significativamente el acceso de personas que no hablan inglés con fluidez. Como resultado, las comunidades de habla hispana a menudo enfrentan desafíos considerables para acceder a oportunidades de capacitación en software. Esta disparidad conduce a desigualdades en la distribución y utilización de las tecnologías científicas, lo que es particularmente preocupante dada la creciente importancia de las habilidades digitales en la actualidad. Para mitigar estos desafíos y promover la inclusión, proponemos realizar un taller de programación en español durante la conferencia. Esta iniciativa tiene como objetivo cerrar la brecha brindando a los participantes de habla hispana igualdad de oportunidades para interactuar y beneficiarse de los avances tecnológicos. Al hacerlo, no sólo mejoramos las capacidades individuales sino que también contribuimos a una distribución más equitativa de los recursos educativos en la comunidad científica. Este taller equipará a los asistentes con habilidades básicas en R. Nuestro objetivo principal es familiarizar a los participantes con RStudio y sus características clave para generar informes reproducibles. Guiaremos a los asistentes a través del proceso de creación y gestión de proyectos en RStudio y les presentaremos la creación de manuscritos reproducibles utilizando documentos Quarto. El taller utilizará un conjunto de datos disponible públicamente de los CDC, que contiene información sobre el uso de drogas y la ideación suicida entre adolescentes, como un ejemplo práctico del uso de R para la investigación académica en salud pública. Explicaremos cómo utilizar funciones como filtrar, mutar, resumir y seleccionar del conjunto de paquetes tidyverse. Concluiremos demostrando cómo usar ggplot2 para crear visualizaciones en R. Al final del taller, los participantes habrán creado un documento reproducible en formato HTML, detallando los pasos de limpieza de datos y el análisis de un problema social contemporáneo importante. Esta presentación tiene como objetivo cerrar la brecha en la alfabetización en programación entre los investigadores de habla hispana y promover métodos para la investigación científica reproducible.</p>
</section>
<section id="speakers-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="speakers-on-the-web">Speakers on the Web</h3>
<p><img src="images/catalina2.png" class="img-fluid"> Catalina Cañizares can be found leading an R club focused on Public Health and Social work <a href="https://ccani007.github.io/R_club/">here</a>, and creating many R teaching resources which can be found <a href="https://ccani007.github.io/ccani_website/teaching.html">here</a>.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="shannon-pileggi" class="level3">
<h3 class="anchored" data-anchor-id="shannon-pileggi">Shannon Pileggi</h3>
<p><img src="images/shannon_pileggi.jpeg" class="img-fluid"></p>
</section>
<section id="biography-4" class="level3">
<h3 class="anchored" data-anchor-id="biography-4">Biography</h3>
<p><a href="https://www.pipinghotdata.com/">Shannon Pileggi</a> is a former statistics college professor now working in industry. I love wrangling, analyzing, and visualizing data, and helping others do the same. Shannon is a member of the leadership team for R-Ladies Global and works as the Lead Data Scientist for the Prostate Cancer Clinical Trials Consortium (PCCTCC).</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="project-oriented-workflows" class="level2">
<h2 class="anchored" data-anchor-id="project-oriented-workflows">Project Oriented Workflows</h2>
<section id="abstract-5" class="level3">
<h3 class="anchored" data-anchor-id="abstract-5">Abstract</h3>
<p>In this workshop, we will establish the concept of and apply best practices to RStudio projects as a basic organizational unit of work. This includes creating robust file paths that travel well in time and space; constructing human and machine readable file names that sort nicely; differentiating workflow elements, analysis inputs, and analysis outputs in project structure; and restarting R frequently, with a blank slate. Upon completion of this workshop, you will have strategies for creating organized and reproducible data analysis projects.</p>
</section>
<section id="shannon-pileggi-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="shannon-pileggi-on-the-web">Shannon Pileggi on the Web</h3>
<p>Shannon is a former statistics professor now working in industry with the The Prostate Cancer Clinical Trials Consortium (PCCTC). I love wrangling, analyzing, and visualizing data, and helping others do the same. Shannon is a member of R-Ladies Global in Pennsylvania, and can be found blogging at <a href="https://www.pipinghotdata.com/">Piping Hot Data</a>, on Mastodon <a href="https://fosstodon.org/@PipingHotData">fosstodon.org/<span class="citation" data-cites="PipingHotData">@PipingHotData</span></a>, and carefully labeling all her <a href="https://www.pipinghotdata.com/posts/2022-09-13-the-case-for-variable-labels-in-r/">data</a>.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="mitchell-ohara-wild" class="level3">
<h3 class="anchored" data-anchor-id="mitchell-ohara-wild">Mitchell O’Hara-Wild</h3>
<p><img src="images/mitchell_ow.jpeg" class="img-fluid"></p>
</section>
<section id="biography-5" class="level3">
<h3 class="anchored" data-anchor-id="biography-5">Biography</h3>
<p><a href="https://mitchelloharawild.com/">Mitchell O’Hara-Wild</a> is a PhD candidate at Monash University and a Data Scientist at Nectric in Clayton, Victoria, Australia. Data scientist and developer of statistical software. Mitchell is our first antipodean workshop leader, and our first to do an evening (very early morning in Australia) workshop. He is a Data scientist and rstats package developer of time series tools, and is addicted to using local IoT devices for monitoring and automating home permaculture.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="tidy-time-series" class="level2">
<h2 class="anchored" data-anchor-id="tidy-time-series">Tidy Time Series</h2>
<section id="abstract-6" class="level3">
<h3 class="anchored" data-anchor-id="abstract-6">Abstract</h3>
<p>This workshop provides a practical introduction to time series analytics and forecasting using R, utilising the tidyverse and tidy time series tools to enable analysis across many time series. Attendees will learn about commonly seen time series patterns, and how to find them with specialised time series graphics created with ggplot2. Then we will use fable to capture these patterns with statistical time series models, and produce probabilistic forecasts. Finally, participants will learn how to evaluate model performance, using accuracy statistics to choose the model which produces the most accurate forecasts. With a combination of foundational concepts and practical demonstrations, this workshop equips participants with the skills to extract meaning from time series data for informed decision-making in various domains.</p>
</section>
<section id="mitchell-ohara-wild-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="mitchell-ohara-wild-on-the-web">Mitchell O’Hara-Wild on the Web</h3>
<p>Mitchell is a Monash University PhD candidate with passions for time series data and data-driven permaculture. Mitchell can be found blogging at https://mitchelloharawild.com/blog.html, working as a Data Scientist at Nectric on <a href="https://github.com/mitchelloharawild/">Github</a>, and optimizing his home garden to surpass Lisa Lendway’s <a href="https://stattlc.com/2021/12/01/digging-into-data-science-using-the-garden-dataset/">crop yields</a> .</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="theo-roe-and-colin-gillespie" class="level3">
<h3 class="anchored" data-anchor-id="theo-roe-and-colin-gillespie">Theo Roe and Colin Gillespie</h3>
<p><img src="images/theo-roe.webp" class="img-fluid"></p>
</section>
<section id="biographies-1" class="level3">
<h3 class="anchored" data-anchor-id="biographies-1">Biographies</h3>
<p><a href="https://www.jumpingrivers.com/authors/theo-roe/">Theo Roe</a> holds a 1st Class Honours MMathStat in Mathematics &amp; Statistics from Newcastle University. He is the author of many of the Jumping Rivers courses and works with a range of clients at Jumping Rivers.</p>
<p><img src="images/colin.jpeg" class="img-fluid"></p>
<p><a href="https://www.jumpingrivers.com/authors/colin-gillespie/">Colin Gillespie</a> is a co-founder and CTO of Jumping Rivers. Colin has been using R since 1999. He’s the author of a number of R packages and has published the book <em>Efficient R Programming</em> with O’Reilly. Colin is a Senior Lecturer at Newcastle University, and completed his Ph.D.&nbsp;in Statistics at the University of Strathclyde.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="from-r-to-powerpoint-advanced-powerpoint-presentations-using-officer" class="level2">
<h2 class="anchored" data-anchor-id="from-r-to-powerpoint-advanced-powerpoint-presentations-using-officer">From R to PowerPoint: Advanced PowerPoint Presentations Using {officer}</h2>
<section id="abstract-7" class="level3">
<h3 class="anchored" data-anchor-id="abstract-7">Abstract</h3>
<p>Imagine you’re asked to recreate a beautiful, complex PowerPoint presentation using R. Despite Quarto’s wide-reaching superpowers, it only offers six measly layout options for Microsoft PowerPoint. The most complex being the simple “two columns”! So, what should I do?</p>
<p>The answer is not Quarto at all; it’s {officer}! In this workshop, we’ll show the process of recreating complex PowerPoint layouts, using {officer}, whilst addressing the pros and cons of abandoning Quarto.</p>
<p>By the end of this workshop, participants will be able to: - Understand what possibilities lie within the “Officeverse” - Why, for complex PowerPoints, {officer} is better than R Markdown and Quarto - Create complex PowerPoint presentations using {officer} - Understand how to best combine PowerPoint templates with {officer} for the optimal workflow - Understand the issues one will encounter when using {officer}</p>
</section>
<section id="theo-roe-and-colin-gillespie-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="theo-roe-and-colin-gillespie-on-the-web">Theo Roe and Colin Gillespie on the Web</h3>
<p>Theo can be found consulting at Jumping Rivers, blogging <a href="https://www.jumpingrivers.com/authors/theo-roe/">here</a> and leaping over the Tyne on Saturday nights in the summertime.</p>
<p>Colin can be found blogging about <a href="https://www.jumpingrivers.com/authors/colin-gillespie/">R</a>, sharing on <a href="https://github.com/csgillespie">Github</a> and consulting as a member of the Royal Statistical Society.</p>
</section>
</section>
</div>
</div>
<div class="columns">
<div class="column" style="width:45%;">
<section id="rami-krispin" class="level3">
<h3 class="anchored" data-anchor-id="rami-krispin">Rami Krispin</h3>
<p><img src="images/rami.jpeg" class="img-fluid"></p>
</section>
<section id="biography-6" class="level3">
<h3 class="anchored" data-anchor-id="biography-6">Biography</h3>
<p><a href="https://www.linkedin.com/in/rami-krispin/">Rami Krispin</a> is a Senior Manager for Data Science and Engineering at Apple. He leads the data science and engineering team at Apple Finance - Services and Infra. The team focuses on translating business problems and requirements into data products using advanced statistical and machine learning models to support and help Apple leadership make data-driven decisions. Rami is also an open-source contributor and the author of <em>Hands-On Time Series Analysis with R</em> (Packt), several R packages for time series analysis, and machine learning applications. Rami is currently working on his next book, <em>Applied Time Series Analysis and Forecasting</em>, which focuses on forecasting at scale.</p>
</section>
</div><div class="column" style="width:10%;">
<!-- empty column to create gap -->
</div><div class="column" style="width:45%;">
<section id="setting-up-a-dockerized-r-development-environment-with-vscode" class="level2">
<h2 class="anchored" data-anchor-id="setting-up-a-dockerized-r-development-environment-with-vscode">Setting Up a Dockerized R Development Environment with VScode</h2>
<section id="abstract-8" class="level3">
<h3 class="anchored" data-anchor-id="abstract-8">Abstract</h3>
<p>VScode enables a seamless integration of Docker with the Dev Containers extension. This tutorial will focus on setting up a dockerized R development environment within VScode using the Dev Container extension and Docker. The tutorial will cover the following topics: - The Foundation of Docker - Rocking R on Docker - The Dev Container extension - Setting Up the R environment</p>
<p>The workshop is based on this tutorial: https://github.com/RamiKrispin/vscode-r</p>
</section>
<section id="rami-krispin-on-the-web" class="level3">
<h3 class="anchored" data-anchor-id="rami-krispin-on-the-web">Rami Krispin on the Web</h3>
<p>Rami can be found blogging about Docker, Time, and Forecasting on <a href="https://medium.com/@rami.krispin">Medium</a>. He can often be found optimizing one of his 123 repositories on <a href="https://github.com/RamiKrispin">Github</a>. Rami studied Economics and Actuarial Science at the University of Michigan, and worked in Data Science in Michigan until 2018, when he gave up his snow shovel and moved to California.</p>
</section>
</section>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/RConsortium\.github\.io\/RMedicine_website\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>